lib/screens/creator_hub/widgets/nfts_list_tile.dart

```diff
@@ -165,6 +165,7 @@ class NftTypeBuilder extends StatelessWidget {
       case AssetType.Image:
         return onImage(context);
       case AssetType.Video:
+      case AssetType.Pdf:
```

Comment:
Why video and audio has same callback? They should be separate.

Reply:
bz they have same ui  to build